src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -486,6 +480,13 @@ export class HaAutomationEditor extends KeyboardShortcutMixin(LitElement) {
     fireEvent(this, "hass-more-info", { entityId: this._entityId });
   }
 
+  private async _showTrace() {
+    if (this._config?.id) {
+      await this.confirmUnsavedChanged();
```

Comment:
What if I dismiss this? Then I will still be navigated to traces and loose my changes...

Reply:
Or we never resolve the promise then? That is not good... we should either reject it, or better just return a boolean

Reply:
Returning boolean is better I think 🙂